dig/default.go

```diff
@@ -20,30 +20,30 @@
 
 package dig
 
-// Default graph is used for all the top-level inject calls
-var defaultGraph = newGraph()
+// DefaultGraph is used for all the top-level inject calls
+var DefaultGraph Graph = newGraph()
```

Comment:
aren't the five global functions(Register, Resolve etc) already exposed all methods defaultGraph interface can use?